announcements/_posts/2020-09-04-w03-networking.md

```diff
@@ -0,0 +1,28 @@
+---
+author: Tim Zabel (Tjzabel)
+title: "Week #3: Networking"
+layout: post-event
+date-start: "2020-09-04 16:30"
+date-end: "2020-09-04 18:00"
+location: "Discord"
+---
+
+Hey RITlug!
+
+For this week's meeting, we will have a recent RIT graduate, Nathaniel Larrimore, who will be giving a presention on Linux networking!
```

Comment:
"presentation". It is not me this time :)

Reply:
presention sounds better